### PR TITLE
good-mb-copyright

### PR DESCRIPTION
**Pull request by AndriyRubanskui**

### CODE REVIEW

File: client/src/components/Copyright/index.jsx [hunk 1]
```diff
@@ -0,0 +1,14 @@
+import React from 'react';
+import './style.scss';
+
+const Copyright = () => (
+        <div className='cop-container'>
```
Comment (anna-kuzii) [inline client/src/components/Copyright/index.jsx]: cop === policeman

use better class name

File: client/src/components/Copyright/index.jsx [hunk 2]
```diff
@@ -0,0 +1,14 @@
...
+          <h2>Sport News</h2>
+          <div>
+          <h4>Copyright ©2019 Sport News</h4>
+          <h4 className='privacy'>Privacy/Terms</h4>
+          </div>
+        </div>
+  );
+
+  export default Copyright;
```
Comment (anna-kuzii) [inline client/src/components/Copyright/index.jsx]: not  here

File: client/src/components/Copyright/style.scss [hunk 3]
```diff
@@ -0,0 +1,29 @@
...
+  display: flex;
+  align-items: center;
+  justify-content: space-between;
+  width: 100%;
+  height: 80px;
+  padding: 25px 55px;
+  color: $rose-white;
+  background-color: $alizarin-crimson;
+
+  & h2 {
```
Comment (anna-kuzii) [inline client/src/components/Copyright/style.scss]: rm &

File: client/src/components/Copyright/style.scss [hunk 4]
```diff
@@ -0,0 +1,29 @@
...
+    font-size: 22px;
+    font-weight: 600;
+    line-height: 28px;
+  }
+
+  & h4 {
```
Comment (anna-kuzii) [inline client/src/components/Copyright/style.scss]: same

File: client/src/components/Copyright/style.scss [hunk 5]
```diff
@@ -0,0 +1,29 @@
...
+    margin-bottom: 0;
+    font-size: 12px;
+    font-weight: 600;
+    line-height: 28px;
+  }
+
+  & .privacy {
```
Comment (anna-kuzii) [inline client/src/components/Copyright/style.scss]: same

File: client/src/components/Copyright/index.jsx (hunk 1 above)
Comment (AndriyRubanskui) [reply client/src/components/Copyright/index.jsx]: funny)

File: client/src/components/Copyright/index.jsx (hunk 2 above)
Comment (AndriyRubanskui) [reply client/src/components/Copyright/index.jsx]: ok

File: client/src/components/Copyright/style.scss (hunk 3 above)
Comment (AndriyRubanskui) [reply client/src/components/Copyright/style.scss]: ok

File: client/src/components/Copyright/style.scss (hunk 4 above)
Comment (AndriyRubanskui) [reply client/src/components/Copyright/style.scss]: ok

File: client/src/components/Copyright/style.scss (hunk 5 above)
Comment (AndriyRubanskui) [reply client/src/components/Copyright/style.scss]: ok

**Pull request merged**